code/client/src/index.js

```diff
@@ -8,6 +8,16 @@ import { Provider } from 'react-redux'
 import { ConnectedRouter } from 'connected-react-router'
 import * as serviceWorker from './serviceWorker'
 import { initAPI, initBlockchain } from './api'
+import * as Sentry from '@sentry/react'
+import { Integrations } from '@sentry/tracing'
+
+if (process.env.NODE_ENV === 'production') {
```

Comment:
environment variables need to be injected by webpack, which is done by EnvironmentPlugin and already configured. An variable already exists for `DEBUG` so this line can use that. `DEBUG` is already loaded in `config.js` but not yet exported so it would be more organized to load it there and import and refer to `config` here